emcc.py

```diff
@@ -181,6 +181,10 @@ def run():
     print shared.get_llvm_target()
     exit(0)
 
+  elif '--cflags' in sys.argv:
+    print ' '.join(shared.COMPILER_OPTS + ['-emit-llvm', '-c'])
```

Comment:
Yeah, I see arguments both ways, but I think you've convinced me on this, will change.